### DISCUSSION TITLE
uninstalled copilot

### DISCUSSION THREAD

**Discussion by philipdenys** [community/community#48472] [Copilot Conversations]
### Select Topic Area

General

### Body

for me, it wasn't a really productive tool...
75% of the time it was repeating it's self

was a great tool in the beginning, but slowly i stopped using it

**Comment by Kaparrs** [comment]
GitHub CoPilot is still very new and might not work for everyone. As the AI systems advance it'll get better at recognising when to pop-up and when not to disturb a user

**Comment by iamcjwilliams** [comment]
@philipdenys Thank you for your feedback, what development environment did you use?

**Reply by Kaparrs** [reply]
Currently I'm not using any helper plug-in that's similar to Copilot, but ChatGPT is pretty good for little code snippets
